beacon-chain/powchain/log_processing.go

```diff
@@ -191,7 +191,14 @@ func (s *Service) ProcessDepositLog(ctx context.Context, depositLog gethTypes.Lo
 		if !s.chainStartData.Chainstarted {
 			deposits := len(s.chainStartData.ChainstartDeposits)
 			if deposits%512 == 0 {
-				log.WithField("deposits", deposits).Info("Processing deposits from Ethereum 1 chain")
+				valCount, err := helpers.ActiveValidatorCount(s.preGenesisState, 0)
```

Comment:
isn't pre genesis state always the same?

Reply:
no its not, it is updated with every deposit